WIP: vaccination-blog-1 Vaccination Scheduling Blog part 1

emmurphy1 opened the pull request:
[HTML draft](http://file.ork.redhat.com/~emmurphy/vaccination-blog/)

emmurphy1 commented:
Please help me correct the image syntax and any other meta-data issues.

ge0ffrey commented on blog/2021-03-01-OptimizingCovid19VaccinationScheduling.adoc:
```diff
@@ -0,0 +1,145 @@
+= Optimizing COVID-19 vaccination appointment scheduling
+:page-interpolate: true
+//:imagesdir: 2021/03/01
+:awestruct-author: pbrown and emmurphy
```
This won't work unfortunately :( Our current system only supports one author (yes, it's unfair, yes we need to fix it, but we can't right now).

Suggestion: just mention "pbrown" only and in the text at the bottom add "Co-authored by Emily Murphy". Reverse it for the second blog.

ge0ffrey commented on blog/2021-03-01-OptimizingCovid19VaccinationScheduling.adoc:
```diff
@@ -0,0 +1,145 @@
+:awestruct-author: pbrown and emmurphy
+//not sure if can have two authors. If not, we'll make it Paul.
+:awestruct-layout: blogPostBase
+:awestruct-tags:
```
```suggestion
:awestruct-tags: [production, vaccination scheduling]
```

triceo commented on blog/2021-03-01-OptimizingCovid19VaccinationScheduling.adoc:
```diff
@@ -0,0 +1,145 @@
+=== The solver
+At the core of OptaPlanner is the solver,  the engine that takes the problem data set and overlays the planning constraints and configurations. The problem data set includes all of the information about the people, the vaccines, and the vaccination centers. The solver works through the various combinations of data and eventually determines an optimized appointment schedule with people assigned to ...
+
+image::2021/03/01/vaccination-scheduling.png[]
```
I looked at other blog posts and they seem to use the following syntax:

     image::JDK11andJDK15UsingParallelGC.png[]

This looks like black magic, but apparently the blog figures out the 2021/03/01 prefix from the title of this file. Strange.

Also, weApparentlyUseCamelCase for image names.

ge0ffrey commented on blog/2021-03-01-OptimizingCovid19VaccinationScheduling.adoc:
```diff
@@ -0,0 +1,145 @@
+//not sure if can have two authors. If not, we'll make it Paul.
+:awestruct-layout: blogPostBase
+:awestruct-tags:
+
```
Make sure there is a twitter/facebook picture by adding this
```suggestion
:awestruct-share_image_filename: vaccination-scheduling.png
```

ge0ffrey replied:
I think the file name will need to change, see below

ge0ffrey replied:
We'll also need to add this line like this one (but with the correct url) as it's being published on RH developers first:
```
:awestruct-canonical_url: "https://developers.redhat.com/blog/2018/02/19/voxxed-days-zurich-2018/"
```

ge0ffrey replied:
Looks like RH developers won't publish first, so removing this canonical url.

triceo commented on blog/2021-03-01-OptimizingCovid19VaccinationScheduling.adoc:
```diff
@@ -0,0 +1,145 @@
+= Optimizing COVID-19 vaccination appointment scheduling
+:page-interpolate: true
+//:imagesdir: 2021/03/01
+:awestruct-author: pbrown and emmurphy
+//not sure if can have two authors. If not, we'll make it Paul.
```
Also, your username is `emmurphy1` and Paul's is `pauljamesbrown`.
See https://github.com/kiegroup/optaplanner-website/blob/master/_config/users.yml

ge0ffrey commented on blog/2021-03-01-OptimizingCovid19VaccinationScheduling.adoc:
```diff
@@ -0,0 +1,145 @@
+=== The solver
+At the core of OptaPlanner is the solver,  the engine that takes the problem data set and overlays the planning constraints and configurations. The problem data set includes all of the information about the people, the vaccines, and the vaccination centers. The solver works through the various combinations of data and eventually determines an optimized appointment schedule with people assigned to ...
+
+image::2021/03/01/vaccination-scheduling.png[]
```
Use the same filenames as they are in optaplanner-docs.

ge0ffrey replied:
Don't prepend the date, see the other blog adocs. That breaks rendering

ge0ffrey commented on blog/2021-03-01-OptimizingCovid19VaccinationScheduling.adoc:
```diff
@@ -0,0 +1,145 @@
+== Continuous planning
+Continuous planning is the technique of managing one or more upcoming planning periods at the same time and repeating that process monthly, weekly, daily, hourly, or even more frequently. The planning window advances incrementally by a specified interval. The following illustration shows a two week planning window that is updated daily:
+
+image::2021/03/01/continuous-planning.png[]
```
Use the same filenames as they are in optaplanner-docs.

ge0ffrey commented on blog/2021-03-01-OptimizingCovid19VaccinationScheduling.adoc:
```diff
@@ -0,0 +1,145 @@
+= Optimizing COVID-19 vaccination appointment scheduling
+:page-interpolate: true
+//:imagesdir: 2021/03/01
```
Add this in, like in the other blogs. That breaks rendering otherwise

ge0ffrey commented:
@emmurphy1 Ideally you can run "rake clean build" on your computer and look at the actual result that we be published in "_site/blog/index.html"

However, that does require to set up Awestruct etc according the instructions in this read me:
https://github.com/kiegroup/optaplanner-website#on-fedora-31
Which can be challenging. Would you be open to trying that out? We're happy to help if you get stuck.

emmurphy1 commented:
@triceo @ge0ffrey I made all of the changes. Here is a summary of the main changes since Geoffrey reviewed:
Replaced injections with vaccination
Rephrased 'second dosers' 
Replaced we with you
Updated the table contents
Changed two of the major headings

Geoffrey, I would like to install Awstruck but don't have time today so if someone else on the team can preview I would appreciate.

Lukas, I wasn't sure about the co-author - should it be my name or tag so included the tag.

emmurphy1 commented:
Also, I left the comments open so you can verify but all have been addressed.
